conans/test/functional/command/profile_test.py

```diff
@@ -131,6 +136,18 @@ def test_profile_update_and_get(self):
         client.run("profile get env.OneMyEnv ./MyProfile")
         self.assertEqual(client.out, "MYVALUe\n")
 
+        client.run("profile update conf.tools.ninja:jobs=10 ./MyProfile")
+        self.assertIn("Package:OtherOption=23", load(pr_path))
```

Comment:
WHy checking an option instead of the conf?

Reply:
My bad 🤦